iOS_SDK/OneSignalSDK/OneSignalUser/Source/OneSignalUserManagerImpl.swift

```diff
@@ -386,6 +392,19 @@ public class OneSignalUserManagerImpl: NSObject, OneSignalUserManager {
             identityModel: identityModel
         )
     }
+    
+    
+    private func fireJwtExpired() {
```

Comment:
JWT jwt and Jwt is used I suggest removing Jwt and sticking with the other two

Reply:
@fhboswell there is jwt and Jwt being used as we are using camelcase. We could use JWT, but probably a bit easier to read as camelcase.

Reply:
@jkasten2 
Jwt if fine if it is preferred however I think we should be consistent. We use JWT in OSUserRequest as part of this PR.

` func addJWTHeader(identityModel: OSIdentityModel) {`

Reply:
Ah I see, yes we should address that in another PR @emawby @nan-li